hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -168,24 +185,24 @@ public void updateMetricsForEmptyData(String actionType) {
       // No-op if metrics are not of type PROMETHEUS_PUSHGATEWAY.
       return;
     }
-    metrics.registerGauge(getMetricsName(actionType, "totalPartitionsWritten"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalFilesInsert"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalFilesUpdate"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalRecordsWritten"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalUpdateRecordsWritten"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalInsertRecordsWritten"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalRecordsDeleted"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalBytesWritten"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalScanTime"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalCreateTime"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalUpsertTime"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalCompactedRecordsUpdated"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalLogFilesCompacted"), 0);
-    metrics.registerGauge(getMetricsName(actionType, "totalLogFilesSize"), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_PARTITIONS_WRITTEN_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_FILES_INSERT_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_FILES_UPDATE_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_RECORDS_WRITTEN_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_UPDATE_RECORDS_WRITTEN_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_INSERT_RECORDS_WRITTEN_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_RECORDS_DELETED), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_BYTES_WRITTEN_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_SCAN_TIME_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_CREATE_TIME_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_UPSERT_TIME_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_COMPACTED_RECORDS_UPDATED_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_LOG_FILES_COMPACTED_STR), 0);
+    metrics.registerGauge(getMetricsName(actionType, TOTAL_LOG_FILES_SIZE_STR), 0);
   }
 
   public void updateCommitMetrics(long commitEpochTimeInMs, long durationInMs, HoodieCommitMetadata metadata,
-      String actionType) {
+      String actionType, boolean enableCompactionLogBlockMetrics) {
```

Comment:
We already have the `HoodiwWriteConfig` as a member of this class, so is there a need to add this boolean param?